src/WorkerProcess/AppScraper.ts

```diff
@@ -36,7 +36,7 @@ export class AppScraper implements Converter {
     #currentRequestUrlGlob: string
     #interceptedResponseHtml: ConverterResult | null
 
-    private constructor(
+    protected constructor(
```

Comment:
Switched from `private` to `protected` here because I have introduced a mock class in this PR which extends from this one. A protected method can be overridden by an extending class, but a private method cannot.